src/components/Editorial/Ctas.tsx

```diff
@@ -0,0 +1,44 @@
+import Button, { type ButtonProps } from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+import { type CommonProps } from 'types/components';
+
+interface CtaButton {
+  text: string;
```

Comment:
A button must have also an `href` or a function to manage an `onClick` 
> href?: string;
> onClick?: Function

Reply:
I think that, to avoid to redefine common props for a button (as href, onClick, etc...), it's much better to use the Button Props interface exposed by mui.

```
...
import { type ButtonProps } from '@mui/material';
...
interface CtaButton extends Partial<ButtonProps> {
  text: string;
}
...
`

Reply:
Yes sorry, forgot to mention that I did it this way to define just a placeholder button, expecting we will have to make a more generic button component available to use in all the components who needs it. I can make it and include in this pr if youd'like or just make a functioning button for this component